### ISSUE TITLE
Where is my ETH gone?

### ISSUE THREAD

**Issue by nevillepettersson** [paritytech/parity#6558]
_Before filing a new issue, please **provide the following information**._

> I'm running:
>
> - **Parity version**: 1.7.2
> - **Operating system**: Windows 7
> - **And installed**: via installer

_Your issue description goes here below. Try to include **actual** vs. **expected behavior** and **steps to reproduce** the issue._

---

I transferred some ETH from my coinbase to ma Parity address:

https://etherscan.io/tx/0xe6ec3eb65a27ec1e13443e91f78eac17ba54538aca58b2d901dde4fd32c7ebd2

I had some trouble getting my Parity Wallet working for almost a day, but now that I'm in my ETH is not showing. All the addresses are correct.

It's been over 20hrs now. What can I do?

Node Health says: Your node is still syncing, the values you see might be outdated. Wait until it's fully synced. I have high-speed fiber, how long does this take and will my ETH show up once this is done?

**Comment by 5chdn** [comment]
Have you tried to restart your node? Warp sync usually takes only 15 minutes.

You are not fully synchronized, that's why your balance is not displayed correctly. Which block are you on? Does the footer say "Warp Restore"?

In addition, please go to the status tab and scroll down and enable the logs, and please, share them here, I will have a look at it.

Are you on HDD or SSD? How much RAM do you have available?

**Comment by nevillepettersson** [comment]
Thanks, I left it on overnight and it’s now caught up.

 

My balance is showing now J

 

Thanks

 

Neville Pettersson

(06) 6511 384

021 08767132
